src/pages/about.jsx

```diff
@@ -97,10 +101,25 @@ const AboutPage = () => {
           </EyeBrowHeaderSection>
           <ButtonGroup>
             <Button center={'center'} text={'Volunteer With Us'}></Button>
-            <Button center={'center'} text={'Partner With Us'}></Button>
+            <Button secondary center={'center'} text={'Partner With Us'}></Button>
           </ButtonGroup>
         </EyebrowHeaderGroup>
       </FAQSection>
+      <Team
```

Comment:
Please look up the "Heading" component and see if it's doing the same thing this component is doing. If it is, let's use that instead. 

However, if you have improved that component in any way, let's incorporate those improvements. 

(for example if it doesn't use React children, let's add it)

And take a look at the styles as well.

Reply:
And so basically you'll want to cmd + shift + F and search for `import Heading` and there were 4 locations. You'll want to delete the link and text url props. Then replace the description prop with a wrapped `p` tag. 

But of course let me know if you have any follow up questions.

Reply:
I realized that adding a background prop to the Heading component only changed the background of the component and it has a max-width of 35...rem. I wasn't large enough to fit the about page. I added a Team wrapper and set the background on it. For some reason the CSS global variables aren't working (still looking into that).